pkg/utility/grouppermissions.go

```diff
@@ -35,7 +35,7 @@ func GetClusterRoleBindingsForGroupPermissions(groupPermissions []api.GroupPermi
 		for _, clusterPermission := range groupPermission.Spec.ClusterPermissions {
 			crb := rbacv1.ClusterRoleBinding{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: groupPermission.Spec.GroupName + "-" + clusterPermission.ClusterRoleName,
+					Name: groupPermission.Spec.GroupName + "-" + clusterPermission,
```

Comment:
design doc says clusterRoleBindingName is "clusterRoleName-GroupName". This contradicts with that. Order matter here?

Reply:
@sam-nguyen7 `clusterPermission` should be a string name of the permission, instead of having to go through the  cstom struct's `ClusterRoleName` attribute. Maybe the variable name should be clarified?

Reply:
Regarding order as documented, it doesn't matter.  We just need a unique name that's consistent w/ docs.  If you want to update this or design doc I don't have a preference.